lib/AST/DeclContext.cpp

```diff
@@ -684,6 +684,7 @@ DeclRange IterableDeclContext::getCurrentMembersWithoutLoading() const {
 }
 
 DeclRange IterableDeclContext::getMembers() const {
+  getASTContext().parseMembers(const_cast<IterableDeclContext*>(this));
   loadAllMembers();
```

Comment:
Can the parsing be done in loadAllMembers()? I imagine right now loadAllMembers() only does something for serialized or imported contexts